app/src/androidTest/java/com/sdp/movemeet/view/home/LoginActivityTest.java

```diff
@@ -35,6 +36,14 @@
     @Rule
     public ActivityScenarioRule<LoginActivity> LoginTestRule = new ActivityScenarioRule<>(LoginActivity.class);
 
+    @Before
+    public void signOutBeforeTest() {
```

Comment:
What is the goal of this test ?

Reply:
It's not a test, it's simply a function that runs before each test and makes sure there is no active user before the tests are run. I was having issues with sign-ins in previous tests messing up these ones, so I added this here.